bin/main.rb

```diff
@@ -1,60 +1,9 @@
 # !/usr/bin/env ruby
-# rubocop: disable Metrics/BlockLength, Metrics/CyclomaticComplexity
-# rubocop: disable Metrics/PerceivedComplexity, Metrics/BlockNesting
-
-def print_board(first_player, second_player)
-  board = ''
-  (0..8).each do |i|
-    board += if first_player.any?(i + 1)
-               ((i + 1) % 3).positive? ? 'X | ' : 'X'
-             elsif second_player.any?(i + 1)
-               ((i + 1) % 3).positive? ? 'O | ' : 'O'
-             else
-               ((i + 1) % 3).positive? ? '  | ' : ' '
-             end
-    board += "\n—————————\n" if ((i + 1) % 3).zero? && i < 8
-  end
-  puts board
-end
-
-def valid_movement?(first_player, second_player, new_movement)
-  begin
-    is_integer = true if Integer(new_movement)
-  rescue StandardError
-    is_integer = false
-  end
-
-  if is_integer
-    if new_movement.to_i >= 1 && new_movement.to_i <= 9
-      return first_player.none?(new_movement) && second_player.none?(new_movement)
-    end
-  end
-  false
-end
-
-def new_movement(player, movement)
-  player.push(movement.to_i)
-end
-
-def player_win?(player_movements)
-  # ONLY FOR TEST, IT IS NOT THE FINAL METHOD TO DETECT IF A PLAYER WON
-  return false if player_movements.length < 4
-
-  [true, false].sample
-end
-
-def game_finished?(first_player_movements, second_player_movements)
-  return 1 if player_win?(first_player_movements)
-  return 2 if player_win?(second_player_movements)
 
-  if first_player_movements.length + second_player_movements.length == 9
-    0
-  else
-    -1
-  end
-end
+require './lib/player.rb'
+require './lib/board.rb'
+require './lib/game.rb'
```

Comment:
[OPTIONAL]
Consider using `require_relative` for files in the project.
Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.

Reply:
Done!. Thank you.